tests/rust-integration-tests/runtimetest/src/main.rs

```diff
@@ -1,9 +1,29 @@
 mod tests;
 mod utils;
 
+use clap::Parser;
 use oci_spec::runtime::Spec;
 use std::path::PathBuf;
 
+#[derive(Parser, Debug)]
+#[clap(version = "0.0.1", author = "youki team")]
+struct Opts {
+    #[clap(subcommand)]
+    command: SubCommand,
+}
+
+#[derive(Parser, Debug)]
+enum SubCommand {
+    #[clap(name = "readonly_paths")]
+    ReadonlyPaths,
+
+    #[clap(name = "set_host_name")]
+    SetHostName,
+
+    #[clap(name = "mounts_recursive")]
+    MountsRecursive,
+}
```

Comment:
What is the reason for adding this? I'm not saying this is a bad idea, but why do we need this?
Also, rather than adding clap just to parse this, how about , for not, `match` on argv[1], and directly compare with string? That way we won't need this whole library.

Reply:
>What is the reason for adding this? I'm not saying this is a bad idea, but why do we need this?

As the number of test items increases, the number of functions executed in the container also increases. I added it because I thought it would be better to execute only a specific function in order to finish the test quickly.
If this is not the correct way to test, revert this to its original state.

Reply:
Hey @higuruchi , the idea to run specific function is great,  and we should definitely do it. However, the `runtimetest` binary is statically compiled and links all linking dependencies directly into the binary. Which means if we add clap, all of clap's linking dependencies will be statically compiled into the binary, which will make it larger in size.

As we just need to match the string, and not do any fancy parsing, can we instead just do
```rust
match argv[1]{
"readonly_paths" =>{/*...*/}
"set_host_name" =>{/*...*/}
"mount_recursive" =>{/*...*/}
_ => {panic!(....)}
}
```
That way we won't need clap

Reply:
That's good.
thank you for your advice